templates/enterprise/secondary-nav.html

```diff
@@ -1,35 +1,38 @@
 <nav class="p-navigation--secondary">
   <div class="row">
     <div class="col-12 u-equal-height">
-      <a class="p-navigation--secondary__banner u-hide--nav-threshold-down" href="/enterprise">
-        <h5 class="p-navigation--secondary__logo">
-          产品和服务
-        </h5>
+      <a class="p-navigation--secondary__banner u-hide--nav-threshold-down"
+         href="/enterprise">
+        <h5 class="p-navigation--secondary__logo">产品和服务</h5>
       </a>
       <ul class="breadcrumbs--secondary">
         <li class="breadcrumbs__item">
-          <a class="breadcrumbs__link {% if request.path == '/enterprise' %}p-link--active{% else %}p-link--soft{% endif %}" href="/enterprise">概览</a>
+          <a class="breadcrumbs__link {% if request.path == '/enterprise' %}p-link--active{% else %}p-link--soft{% endif %}"
+             href="/enterprise">概览</a>
         </li>
         <li class="breadcrumbs__item">
           <a class="breadcrumbs__link p-link--soft" href="/support/pro">Ubuntu Pro</a>
         </li>
+        <li class="breadcrumbs__item">
+          <a class="breadcrumbs__link p-link--soft" href="/infrastructure">基础架构</a>
```

Comment:
I don't see this change to navigation being mentioned as the scope of the issue. Did I miss it being requested somewhere?

JIRA issue https://warthogs.atlassian.net/browse/WD-20643 seems to explicitly ask for an isolated page not linked from navigation.